### DISCUSSION TITLE
YARP-enabled Gitpod workspaces available

### DISCUSSION THREAD

**Discussion by pattacini** [robotology/community#459] [Show and tell]
Following up on #458, we have lately streamlined the user experience in dealing with a **YARP-enabled workspace in the cloud** ☁

We have eventually switched to the use of [Gitpod](https://gitpod.io) in place of Codenvy (no longer available).

ℹ Gitpod has rich [documentation](https://www.gitpod.io/docs) you can peruse online.

You can sign up for Gitpod using your **GitHub credentials** having **50 hours/month** of free usage. Don't forget to tune up how Gitpod can access your GitHub through the [Access Control](https://gitpod.io/integrations) dashboard.

Any time you see the **badge** below exposed from within one of our repositories, just click on it to get started with the associated Gitpod workspace.

![Badge](https://gitpod.io/button/open-in-gitpod.svg)

The badge hides the following URL that Gitpod uses to [start up the workspace](https://www.gitpod.io/docs/getting-started):

- https://gitpod.io/#**<repository-url>**

To ease this process, you can also install the [Gitpod browser extensions](https://www.gitpod.io/docs/browser-extension).

Eventually, you'll be redirected to an environment that will look like this for the repo [`vvv-school/assignment_git`](https://github.com/vvv-school/assignment_git) 🎥

https://user-images.githubusercontent.com/3738070/102713911-b22cb200-42cb-11eb-9f3a-ab09f995f8bb.mp4

In a nutshell, you can code with absolutely no installation effort directly in the cloud with a full-fledged Ubuntu system that also sports the **graphic interface X** to run the simulator 🤖 Watch the whole video to know how to achieve that.

▶ Remember that you can easily **start/stop Gitpod workspaces** at will as described [here](https://www.gitpod.io/docs/life-of-workspace) ⏹

Happy coding ✨
